Reply/app/src/main/java/com/example/reply/ui/ReplyApp.kt

```diff
@@ -227,4 +235,75 @@ fun ReplyNavGraph(
             }
         }
     }
-}
\ No newline at end of file
+}
+
+@Composable
+private fun ReplyNavHost(
+    modifier: Modifier = Modifier,
+    navController: NavHostController,
+    contentType: ReplyContentType,
+    replyHomeUIState: ReplyHomeUIState,
+    navigationType: ReplyNavigationType,
+    navigateToDetail: (Long) -> Unit
+) {
+    NavHost(
+        modifier = modifier,
+        navController = navController,
+        startDestination = ReplyRouts.INBOX_ROUTE,
+    ) {
+//        inboxGraph(
+//            nestedGraphs = {
+//                emailDetailGraph {
+//                    ReplyEmailDetail(email = replyHomeUIState.selectedEmail ?: replyHomeUIState.emails.first()) {
+//                        navController.popBackStack()
+//                    }
+//                }
+//            }
+//        ) {
+//            ReplyInboxScreen(contentType, replyHomeUIState, navigationType, navigateToDetail)
```

Comment:
@simona-anomis this UI doesn't react to replyHomeUIState if put this way, below code react to homeUIState chages.